scylla-cql/src/frame/frame_errors.rs

```diff
@@ -39,6 +39,8 @@ pub enum ParseError {
     CqlResultParseError(#[from] CqlResultParseError),
     #[error("Schema change event deserialization failed: {0}")]
     SchemaChangeEventParseError(#[from] SchemaChangeEventParseError),
+    #[error("Failed to deserialize cluster change event: {0}")]
+    ClusterChangeEventParseError(#[from] ClusterChangeEventParseError),
```

Comment:
And here the new event parse error is only added to ParseError, not `CqlResultParseError` - I think it should be this way for all errors.